### PR TITLE
[T2A2][F12-A3]JoelLim

### PR DESCRIPTION
**Pull request by aljorhythm**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -1221,7 +1234,7 @@ private static String getUsageInfoForExitCommand() {
 	 * @return string without the sign
 	 */
 	private static String removePrefixSign(String s, String sign) {
-		return s.replace(sign, "");
+		return removePrefix(s, sign);
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:1237]: this is a nice way to implement the generalization :+1:
you have ensured that the rest of the code is not disturbed. quite a unique attempt :-)

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -1212,6 +1212,19 @@ private static String getUsageInfoForExitCommand() {
...
+	 * Removes prefix from parameter string
+	 * 
+	 * @param fullstring
+	 *            parameter string
+	 * @param prefix
+	 *            prefix to remove from parameter string
+	 */
+
+	private static String removePrefix(String fullstring, String prefix) {
+		return fullstring.replaceFirst("^" + prefix, "");
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:1224]: indentation is off here.
please use 4 spaces for indentation as per the coding guidelines.
Tip: you can get eclipse to do that for you. check out how!

**Comment by okkhoy** [comment]
@funnieguy please pay closer attention to coding standard violations like variable names and method names in the given code base. 

Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.

some comments added. please take a look, ack and close the PR

on an unrelated note, please add a photo to your github profile to help us identify better

**Comment by aljorhythm** [comment]
Acknowledged

**Pull request closed without merging**